cinema/serializers.py

```diff
@@ -1 +1,115 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    Genre,
+    Actor,
+    MovieSession,
+    Order,
+    Ticket,
+    CinemaHall,
+    Movie
+)
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+        read_only_fields = ("id",)
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row", "capacity")
```

Comment:
The `CinemaHallSerializer` includes a `capacity` field. Please verify if the `capacity` field exists in the `CinemaHall` model. If it doesn't exist, remove it from the serializer to prevent potential errors.